### PR TITLE
Add updated babel versions for setup config

### LINKED ISSUE #54: CHECKIN#1: FS - Project Setup
[ ] - Set up your repo/project_board using conveyor belt (include all team members name and 1 email address for notification)
[ ] - Install prerequisites (Node & Postgres)
[ ] - Local setup (follow `Setup and Running` section of the Crate Readme; use Configurations on project setup site)
[ ] - Group Reflection

### PR DESCRIPTION
**Pull request by leahriffell**
Closes #54

### CODE REVIEW

File: code/web/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@babel/core": "7.9.0",
+    "@babel/core": "^7.9.0",
```
Comment (priyapower) [inline code/web/package.json:21]: This was required for some local environments to run this app

File: code/web/package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "webpack-cli": "3.3.11"
   },
   "dependencies": {
+    "@babel/cli": "^7.12.8",
```
Comment (priyapower) [inline code/web/package.json:36]: This dependency is also necessary for some local environments to run this app

**Review by priyapower — APPROVED** [review]
Looks great! I think it is ready to merge!

**Review by judithpillado — APPROVED** [review]

**Comment by leahriffell** [comment]
@judithpillado or @priyapower, could you please merge this one since it's been approved? Thank you! :)

**Pull request merged**